<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="resultsField">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>781</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="oneStepBt">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>One Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runBt">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupPalette">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Palette</string>
    </property>
    <widget class="QPushButton" name="alfabtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>α</string>
     </property>
    </widget>
    <widget class="QPushButton" name="betabtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>β</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gammabtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>γ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deltabtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>δ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="epsilonbtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ε</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dsetabtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ζ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="etabtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>η</string>
     </property>
    </widget>
    <widget class="QPushButton" name="thetabtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>θ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="iotabtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ι</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kappabtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>κ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lambdaminbtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>λ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lambdamaybtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Λ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mybtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>μ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nybtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ν</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xibtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ξ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="omicronbtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ο</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pibtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>π</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rhobtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ρ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sigmabtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>σ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="taubtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>τ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ypsilonbtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>υ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fibtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>φ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="jibtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>χ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="psibtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ψ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="omegabtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ω</string>
     </property>
    </widget>
    <widget class="QPushButton" name="arrowbtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>→  </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>Input</string>
    </property>
    <addaction name="actionSimple_Input"/>
    <addaction name="actionMultiple_Inputs"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPalette"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInput"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSimple_Input">
   <property name="text">
    <string>Simple Input</string>
   </property>
  </action>
  <action name="actionMultiple_Inputs">
   <property name="text">
    <string>Multiple Inputs</string>
   </property>
  </action>
  <action name="actionPalette">
   <property name="text">
    <string>Palette</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
